internal/infrastructure/db/migrations/20221020065955_integration_features_weight.sql

```diff
@@ -0,0 +1,36 @@
+-- +goose Up
+-- +goose StatementBegin
+SELECT 'up SQL query';
+
+ALTER TABLE integration_features ADD feature_weight float NULL;
+BEGIN;
```

Comment:
thought: Goose wraps everything in a transaction, so you may not need to say this.

Reply:
correct, probably not necessary, regardless test without it.